src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onClickAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const { onClickAdd } = props;
+
+  const [searchQuery, setSearchQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorInFinding, setIsErrorInFinding] = useState(false);
+
+  const handleSubmit: React.FormEventHandler<HTMLFormElement> = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    getMovie(searchQuery)
+      .then((response) => {
+        const data = response;
+
+        if ('Error' in data) {
+          setMovie(null);
+          setIsErrorInFinding(true);
```

Comment:
you do the same in your `.catch`
do you need this duplication?

Reply:
deleted `setMovie(null)` & `.catch`